pkg/hive/install.go

```diff
@@ -76,7 +81,8 @@ func (c *clusterManager) Install(ctx context.Context, sub *api.SubscriptionDocum
 	}
 
 	resources := []kruntime.Object{
-		sppSecret,
+		azureCredentialSecret,
+		clusterManifestsSecret,
```

Comment:
Is there an opportunity to combine these two so we only pass one secrets object to Hive?

Reply:
No, it has to be separate secrets as these both secrets go in their dedicated volume for the installer pod.
azureCredentialSecret -> `/.azure`
clusterManifestsSecret -> `/manifests`

Also, the usage for both secrets are different, so it is kept as per what Hive appears.